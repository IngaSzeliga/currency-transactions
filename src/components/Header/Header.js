import React from "react";
import "./Header.scss";

const Header = () => {
  return <h1 className="header-container">CURRENCY TRANSACTIONS</h1>;
};

export default React.memo(Header);
