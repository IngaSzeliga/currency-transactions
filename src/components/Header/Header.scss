@import "../../config/style.scss";
@import "../../config/sizes.scss";

.header-container {
  font-size: 40px;
  color: $light-black;
  @media only screen and (max-width: $tablet) {
    font-size: 25px;
  }
}
