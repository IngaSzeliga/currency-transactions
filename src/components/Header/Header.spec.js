import React from "React";
import { shallow } from "enzyme";
import Header from ".";

describe("Header component", () => {
  const component = shallow(<Header />);
  it("has working component", () => {
    expect(component).toBeDefined();
  });
  it("expects component to never be changed", () => {
    expect(component).toMatchSnapshot();
  });
});
