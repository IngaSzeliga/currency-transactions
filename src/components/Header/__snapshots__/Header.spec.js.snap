// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component expects component to never be changed 1`] = `
<h1
  className="header-container"
>
  CURRENCY TRANSACTIONS
</h1>
`;
